gatsby-node.js

```diff
@@ -145,12 +143,8 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     }
   })
 
-  // TODO move this into graphql
-  posts = posts.sort(
-    (a, b) =>
-      new Date(b.childMdx.frontmatter.date) -
-      new Date(a.childMdx.frontmatter.date)
-  )
+
+  posts = posts.sort((a,b)=> new Date(b.childMdx.frontmatter.date) - new Date(a.childMdx.frontmatter.date));
```

Comment:
These are prettier changes, no need commit. Have a to apply all the prettier changes in one commit later.

Reply:
Nevermind, Dont work on this. We can merge this PR.